File: testing/LatihanInstrumentationTest/java/app/src/main/java/com/dicoding/picodiploma/myunittest/MainViewModel.java
package com.dicoding.picodiploma.myunittest;

class MainViewModel {
    private final CuboidModel cuboidModel;

    MainViewModel(CuboidModel cuboidModel) {
        this.cuboidModel = cuboidModel;
    }

    void save(double l, double w, double h) {
        cuboidModel.save(l, w, h);
    }

    public double getCircumference() {
        return cuboidModel.getCircumference();
    }

    public double getSurfaceArea() {
        return cuboidModel.getSurfaceArea();
    }

    public double getVolume() {
        return cuboidModel.getVolume();
    }
}

File: testing/LatihanInstrumentationTest/kotlin/app/src/main/java/com/dicoding/picodiploma/myunittest/MainViewModel.kt
package com.dicoding.picodiploma.myunittest

internal class MainViewModel(private val cuboidModel: CuboidModel) {

    val circumference: Double
        get() = cuboidModel.circumference

    val surfaceArea: Double
        get() = cuboidModel.surfaceArea

    val volume: Double
        get() = cuboidModel.volume

    fun save(l: Double, w: Double, h: Double) {
        cuboidModel.save(l, w, h)
    }
}
